cities/bexhill.md

```diff
@@ -1,17 +1,15 @@
 ---
-layout: city
+layout: city_online
 lockdown: false
-city_name: Bexhill
-jam_name: Bexhill MathsJam
+city_name: Bexhill (Virtual)
+jam_name: Bexhill Virtual MathsJam
 email: bexhill@mathsjam.com
 organiser:
     name: Simon
     email: bexhill@mathsjam.com
 location:
     group: uk-ireland
-    pub_name: Traffers Bar
-    description: ', on Egerton Road'
-    url: http://www.beerintheevening.com/pubs/s/30/30913/Traffers_Bar/Bexhill_on_sea
+    software: TBC
```

Comment:
@Simon-Allen currently, we've set the software used for the virtual jam to "TBC", so it confusingly says "We usually meet on the second-last Tuesday of the month, from 7pm in the evening UK time, on TBC."

What software are you using to run the jam?